### PR TITLE
Create project level README.md

### PR DESCRIPTION
**Pull request by Rozzii**
This was missing from the project and folks who find us directly via GitHub would be welcomed by this README first.

### CODE REVIEW

**Comment by Rozzii** [comment]
I made this with github GUI I will update it with the DCO.

**Comment by tuminoid** [comment]
For reference, this is relevant documentation: https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Metal³
+
+The Metal³ project (pronounced: "Metal Kubed") provides components for bare
+metal host management with Kubernetes. You can enrol your bare metal machines,
```
Comment (tuminoid) [inline profile/README.md]:
```suggestion
metal host management with Kubernetes. You can enroll your bare metal machines,
```

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+* Automatically remediate failed nodes by rebooting them and removing them from
+  the cluster if necessary
+
+You can even deploy Metal³ to your clusters so that they can manage other
+clusters using Metal³...
+
+Metal³ is [open-source](https://github.com/metal3-io) and welcomes community
+contributions. The community meets at the following venues:
+
+* \#cluster-api-baremetal on [Kubernetes Slack](https://slack.k8s.io/)
```
Comment (tuminoid) [inline profile/README.md]:
```suggestion
* `#cluster-api-baremetal` on [Kubernetes Slack](https://slack.k8s.io/)
```

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+* Metal³ development [mailing list](https://groups.google.com/g/metal3-dev)
```
Comment (tuminoid) [inline profile/README.md]:
```suggestion
* Metal³ [development mailing list](https://groups.google.com/g/metal3-dev)
```

File: profile/README.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+* From the mailing list, you'll also be able to find the details of a weekly
+  Zoom community call on Wednesdays at 14:00 GMT
+* Roadmap discussion: https://docs.google.com/document/d/1Exm5k6JB2JlD9_FOaM68R2ASHi7ZsryF_xEqLG1j4zA
```
Comment (tuminoid) [inline profile/README.md]: I don't think this belongs in readme

File: profile/README.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+* Flake tracker: https://github.com/orgs/metal3-io/projects/4
+* CLO monitor: https://clomonitor.io/projects/cncf/metal3-io 
+* Dev stats: https://metal3.devstats.cncf.io/d/8/dashboards?orgId=1&from=now-1y&to=now-1h&editPanel=2 
+* Container images (Quay): https://quay.io/organization/metal3-io
+* Jenkins periodics: https://jenkins.nordix.org/view/Metal3%20Periodic/ 
+* Prow: https://prow.apps.test.metal3.io/
+* Security processes and volnerability disclousure info: https://book.metal3.io/security_policy
+* Community, governance and contribution policies can be found here: https://github.com/metal3-io/community
+* User guide: https://book.metal3.io/introduction
+* Architectural design proposals: https://github.com/metal3-io/metal3-docs/tree/main/design
```
Comment (tuminoid) [inline profile/README.md]: This list in general needs some better sorting, say
1. communication channels
2. user guides
3. development basic
4. CI 

as is, its a random list.

File: profile/README.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+Metal³ is [open-source](https://github.com/metal3-io) and welcomes community
+contributions. The community meets at the following venues:
+
+* \#cluster-api-baremetal on [Kubernetes Slack](https://slack.k8s.io/)
+* Metal³ development [mailing list](https://groups.google.com/g/metal3-dev)
+* From the mailing list, you'll also be able to find the details of a weekly
+  Zoom community call on Wednesdays at 14:00 GMT
+* Roadmap discussion: https://docs.google.com/document/d/1Exm5k6JB2JlD9_FOaM68R2ASHi7ZsryF_xEqLG1j4zA
+* Roadmap GitHub project: https://github.com/orgs/metal3-io/projects/8
+* Feel free to add your organization as an adopter to  https://github.com/metal3-io/community/blob/main/ADOPTERS.md
```
Comment (tuminoid) [inline profile/README.md]: Adopters is misplaced here, but it should be somewhere in more human like form, like "If you're already using Metal3 in your organization, don't forget to add yourself to adopters!" or something.

File: profile/README.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+You can even deploy Metal³ to your clusters so that they can manage other
+clusters using Metal³...
+
+Metal³ is [open-source](https://github.com/metal3-io) and welcomes community
+contributions. The community meets at the following venues:
+
+* \#cluster-api-baremetal on [Kubernetes Slack](https://slack.k8s.io/)
+* Metal³ development [mailing list](https://groups.google.com/g/metal3-dev)
+* From the mailing list, you'll also be able to find the details of a weekly
```
Comment (tuminoid) [inline profile/README.md]: Can we not just put the details here, its not like they're changing weekly?

**Review by tuminoid — COMMENTED** [review]
Adding an org level readme is great idea!

Some nits, but also would like some other comments what should be included.  I feel this goes into too much details belong a profile readme. If its like personal readme, it'll push down other content in the org page a lot.

File: profile/README.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+* Configure BIOS and RAID settings on your hosts
+* Optionally clean a host's disks as part of provisioning
+* Install and boot an operating system image of your choice
+* Deploy Kubernetes
+* Upgrade Kubernetes or the operating system in your clusters with a
+  non-disruptive rolling strategy
+* Automatically remediate failed nodes by rebooting them and removing them from
+  the cluster if necessary
+
+You can even deploy Metal³ to your clusters so that they can manage other
```
Comment (kashifest) [inline profile/README.md]:
```suggestion
You can even deploy Metal³ to your clusters so that Metal³ can manage other clusters. 
```

**Review by kashifest — COMMENTED** [review]
/approve
@tuminoid has very good reviews already, just address them and its good to go. Thanks for adding it.

**Comment by tuminoid** [comment]
@rozzii would you finish this?

**Comment by Rozzii** [comment]
> @Rozzii would you finish this?

I will soon, when I have some free time.

File: profile/README.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+
+You can even deploy Metal³ to your clusters so that you can manage other
+clusters.
+
+Metal³ is [open-source](https://github.com/metal3-io) and welcomes community
+contributions. The community meets at the following venues:
+
+## Contacts
+* `#cluster-api-baremetal` on [Kubernetes Slack](https://slack.k8s.io/)
+* Metal³ [development [mailing list](https://groups.google.com/g/metal3-dev)
```
Comment (tuminoid) [inline profile/README.md]:
```suggestion
* Metal³ [development mailing list](https://groups.google.com/g/metal3-dev)
```

File: profile/README.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+* Zoom community meeting call on Wednesdays at 14:00 GMT
+
+## CI
+* Prow: https://prow.apps.test.metal3.io/
```
Comment (tuminoid) [inline profile/README.md]: No bare links. Either put them in `<>` if you want the link to be visible, if not then 

```suggestion
* Prow: <https://prow.apps.test.metal3.io/>
```
or
```suggestion
* [Prow](https://prow.apps.test.metal3.io/)
```

File: profile/README.md [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+
+Metal³ is [open-source](https://github.com/metal3-io) and welcomes community
+contributions. The community meets at the following venues:
+
+## Contacts
+* `#cluster-api-baremetal` on [Kubernetes Slack](https://slack.k8s.io/)
+* Metal³ [development [mailing list](https://groups.google.com/g/metal3-dev)
+* Zoom community meeting call on Wednesdays at 14:00 GMT
+
+## CI
```
Comment (tuminoid) [inline profile/README.md:26]:
```suggestion
## CI

```

File: profile/README.md [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+* Automatically remediate failed nodes by rebooting them and removing them from
+  the cluster if necessary
+
+You can even deploy Metal³ to your clusters so that you can manage other
+clusters.
+
+Metal³ is [open-source](https://github.com/metal3-io) and welcomes community
+contributions. The community meets at the following venues:
+
+## Contacts
```
Comment (tuminoid) [inline profile/README.md:18]:
```suggestion
## Contacts

```

File: profile/README.md [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+* `#cluster-api-baremetal` on [Kubernetes Slack](https://slack.k8s.io/)
+* Metal³ [development [mailing list](https://groups.google.com/g/metal3-dev)
+* Zoom community meeting call on Wednesdays at 14:00 GMT
+
+## CI
+* Prow: https://prow.apps.test.metal3.io/
+* Jenkins periodics: https://jenkins.nordix.org/view/Metal3%20Periodic/
```
Comment (tuminoid) [inline profile/README.md]: same for all these links

File: profile/README.md [hunk 14]
```diff
@@ -0,0 +1,65 @@
...
+* Container images (Quay): https://quay.io/organization/metal3-io
+* Flake tracker: https://github.com/orgs/metal3-io/projects/4
+
+## Info
```
Comment (tuminoid) [inline profile/README.md:33]:
```suggestion
## Info

```

File: profile/README.md [hunk 15]
```diff
@@ -0,0 +1,65 @@
...
+* User guide: https://book.metal3.io/introduction
+* Architectural design proposals: https://github.com/metal3-io/metal3-docs/tree/main/design
+* Security processes and volnerability disclousure info: https://book.metal3.io/security_policy
+* Community, governance and contribution policies can be found here: https://github.com/metal3-io/community
+* Roadmap GitHub project: https://github.com/orgs/metal3-io/projects/8
+
+## Stats
```
Comment (tuminoid) [inline profile/README.md:41]:
```suggestion
## Stats

```

**Review by tuminoid — COMMENTED** [review]
We don't have markdown linter here, so I'm doing the nitting it would do.

Dev list link also ha broken parenthesis issue.

File: profile/README.md [hunk 16]
```diff
@@ -0,0 +1,69 @@
...
+clusters.
+
+Metal³ is [open-source](https://github.com/metal3-io) and welcomes community
+contributions. The community meets at the following venues:
+
+## Contacts
+
+* `#cluster-api-baremetal` on [Kubernetes Slack](https://slack.k8s.io/)
+* Metal³ [development mailing list](https://groups.google.com/g/metal3-dev)
+* Zoom community meeting call on Wednesdays at 14:00 UTC
```
Comment (tuminoid) [inline profile/README.md]: This does not have details. Should we link the meeting notes doc here, as it contains all the information to join + history? 

```suggestion
* [Zoom community meeting](https://docs.google.com/document/d/1IkEIh-ffWY3DaNX3aFcAxGbttdEY_symo7WAGmzkWhU) call on Wednesdays at 14:00 UTC
```

**Review by tuminoid — COMMENTED** [review]
Zoom / community meeting info added, then we are good to merge IMO.

**Comment by kashifest** [comment]
ping @Rozzii lets merge this

**Comment by dtantsur** [comment]
This page duplicates a lot of information from the user guide, and I wonder if it should be just a quick description and a bunch of links instead. What are the chances that we'll remember to keep this page up-to-date?

**Comment by Rozzii** [comment]
> This page duplicates a lot of information from the user guide, and I wonder if it should be just a quick description and a bunch of links instead. What are the chances that we'll remember to keep this page up-to-date?

Yeah that sounds good to me, I will make this shorter.

**Comment by Rozzii** [comment]
I will cancel this and I will push the change from nordix.

**Pull request closed without merging**